addons/packages/kapp-controller/0.30.0/bundle/config/values.star

```diff
@@ -8,3 +8,10 @@ if hasattr(values.kappController, 'namespace') and values.kappController.namespa
 else:
     kappNamespace = values.namespace
 end
+
+def generateBashCmdForDNS(coreDNSIP):
+    # This command added the coreDNS IP as the first entry of resolv.conf
```

Comment:
(non-blocking): Would be great if coreDNSIP is checked if valid IP using ytt IP module.